.github/workflows/buildit.yaml

```diff
@@ -1,35 +1,58 @@
 name: BuildIt
+
 on:
-  workflow_dispatch:
+  release:
+    types:
+      - created
+      - prereleased
+      - edited
+      - published
+
 jobs:
   build:
+    permissions: write-all
     runs-on: windows-2022
     steps:
-    - name: "Clone source"
-      uses: actions/checkout@v2
-      with:
-        submodules: 'true'
-    - name: "Setup msvc"
-      uses: ilammy/msvc-dev-cmd@v1
-    - name: "Download Qt5"
-      shell: bash
-      run: |
-        curl --output C:/qt5.7z -L "https://github.com/LoL-Fantome/lolcustomskin-tools/releases/download/release23/qt5.15.5-x86_64-msvc-static.7z"
-        7z x -oC:/ C:/qt5.7z
-    - name: "Build"
-      run: |
-        mkdir build
-        cd build
-        cmake -G "NMake Makefiles" "-DCMAKE_PREFIX_PATH:PATH=C:/qt5.15.5-x86_64-msvc-static" "-DCMAKE_MSVC_RUNTIME_LIBRARY:STRING=MultiThreaded" "-DCMAKE_BUILD_TYPE:STRING=Release" ..
-        cmake --build .
-    - name: "Package"
-      shell: bash
-      run: |
-        ./make-release.sh build
-        7z a cslol-manager-windows.zip cslol-manager/
-    - name: 'Upload Artifact'
-      uses: actions/upload-artifact@v2
-      with:
-        name: cslol-manager-windows
-        path: cslol-manager-windows.zip
-        retention-days: 15
+      - name: "Clone source"
+        uses: actions/checkout@v3
+        with:
+          submodules: 'true'
+          ref: master
+
+      - name: "Setup msvc"
+        uses: ilammy/msvc-dev-cmd@v1
+
+      - name: "Download Qt5"
+        shell: bash
+        run: |
+          curl --output C:/qt5.7z -L "https://github.com/LoL-Fantome/lolcustomskin-tools/releases/download/release23/qt5.15.5-x86_64-msvc-static.7z"
+          7z x -oC:/ C:/qt5.7z
+
+      - name: "Build"
+        run: |
+          mkdir build
+          cd build
+          cmake -G "NMake Makefiles" "-DCMAKE_PREFIX_PATH:PATH=C:/qt5.15.5-x86_64-msvc-static" "-DCMAKE_MSVC_RUNTIME_LIBRARY:STRING=MultiThreaded" "-DCMAKE_BUILD_TYPE:STRING=Release" ..
+          cmake --build .
+
+      - name: "Package"
+        shell: bash
+        run: |
+          ./make-release.sh build
+          7z a cslol-manager-windows.zip cslol-manager/
```

Comment:
I'm thinking we could also put the tag into the name of the artifact.

Reply:
Sorry for the late response, I've been busy with work.

I'll get right on it once I'm home.

Reply:
Can you modify release.yaml instead of buildit.yaml is intended for manual build jobs.
Also if possible, both macos and windows should be part of same release.yaml, otherwise create releasemac.yaml.

Reply:
Do you simply want the current `release.yaml` to be completely rewritten, or?

Reply:
yeah that would be ideal